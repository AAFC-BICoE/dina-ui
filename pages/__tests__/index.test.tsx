import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import IndexPage from "../index";

describe("Index page", () => {
  it("Renders the index page.", () => {
    const wrapper = shallow(<IndexPage />);
    expect(toJson(wrapper)).toMatchSnapshot("Index page shallow render");
  });
});
