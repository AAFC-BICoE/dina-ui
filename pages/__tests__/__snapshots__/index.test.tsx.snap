// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page Renders the index page.: Index page shallow render 1`] = `
<div
  className="jsx-243749409"
>
  <Head
    title="Home"
  />
  <Nav />
  <div
    className="jsx-243749409 hero"
  >
    <h1
      className="jsx-243749409 title"
    >
      Welcome to Next!
    </h1>
    <p
      className="jsx-243749409 description"
    >
      To get started, edit 
      <code
        className="jsx-243749409"
      >
        pages/index.js
      </code>
       and save to reload.
    </p>
    <div
      className="jsx-243749409 row"
    >
      <Link
        href="https://github.com/zeit/next.js#getting-started"
      >
        <a
          className="jsx-243749409 card"
        >
          <h3
            className="jsx-243749409"
          >
            Getting Started →
          </h3>
          <p
            className="jsx-243749409"
          >
            Learn more about Next on Github and in their examples
          </p>
        </a>
      </Link>
      <Link
        href="https://open.segment.com/create-next-app"
      >
        <a
          className="jsx-243749409 card"
        >
          <h3
            className="jsx-243749409"
          >
            Examples →
          </h3>
          <p
            className="jsx-243749409"
          >
            Find other example boilerplates on the
             
            <code
              className="jsx-243749409"
            >
              create-next-app
            </code>
             site
          </p>
        </a>
      </Link>
      <Link
        href="https://github.com/segmentio/create-next-app"
      >
        <a
          className="jsx-243749409 card"
        >
          <h3
            className="jsx-243749409"
          >
            Create Next App →
          </h3>
          <p
            className="jsx-243749409"
          >
            Was this tool helpful? Let us know how we can improve it
          </p>
        </a>
      </Link>
    </div>
  </div>
  <JSXStyle
    css=".hero.jsx-243749409{width:100%;color:#333;}.title.jsx-243749409{margin:0;width:100%;padding-top:80px;line-height:1.15;font-size:48px;}.title.jsx-243749409,.description.jsx-243749409{text-align:center;}.row.jsx-243749409{max-width:880px;margin:80px auto 40px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-box-pack:space-around;-webkit-justify-content:space-around;-ms-flex-pack:space-around;justify-content:space-around;}.card.jsx-243749409{padding:18px 18px 24px;width:220px;text-align:left;-webkit-text-decoration:none;text-decoration:none;color:#434343;border:1px solid #9b9b9b;}.card.jsx-243749409:hover{border-color:#067df7;}.card.jsx-243749409 h3.jsx-243749409{margin:0;color:#067df7;font-size:18px;}.card.jsx-243749409 p.jsx-243749409{margin:0;padding:12px 0 0;font-size:13px;color:#333;}"
    styleId="243749409"
  />
</div>
`;
